contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveUtilities.java

```diff
@@ -398,12 +398,18 @@ public static void addConfToJob(final JobConf job, final Properties properties)
    * Wrapper around {@link MetaStoreUtils#getPartitionMetadata(Partition, Table)} which also adds parameters from table
    * to properties returned by {@link MetaStoreUtils#getPartitionMetadata(Partition, Table)}.
    *
-   * @param partition
-   * @param table
-   * @return
+   * @param partition {@link Partition} instance
+   * @param table {@link Table} instance
+   * @return properties
    */
   public static Properties getPartitionMetadata(final Partition partition, final Table table) {
-    final Properties properties = MetaStoreUtils.getPartitionMetadata(partition, table);
+    final Properties properties;
```

Comment:
I was thinking about adding columns to a list if it is not already in the list. This list should belong to the table, and each partition will refer to an index of this list for columns. Do you think this may be a way to store columns?